oblt-cli/cluster-destroy/README.md

```diff
@@ -0,0 +1,42 @@
+# <!--name-->Oblt-cli destroy cluster<!--/name-->
+<!--description-->
+Run the oblt-cli wrapper to destroy the given cluster
+<!--/description-->
+
+## Inputs
+<!--inputs-->
+| Name                | Description                           | Required | Default |
+|---------------------|---------------------------------------|----------|---------|
+| `cluster-name`      | The cluster name                      | `false`  | ` `     |
+| `cluster-info-file` | The cluster info file (absolute path) | `false`  | ` `     |
+| `github-token`      | The GitHub access token.              | `true`   | ` `     |
+<!--/inputs-->
+
+## Usage
+<!--usage action="elastic/oblt-actions/oblt-cli/cluster-destroy" version="env:VERSION"-->
+```yaml
+---
+name: Create the cluster using oblt-cli and destroy it
+
+...
+
+jobs:
+  create-cluster-and-destroy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: elastic/oblt-actions/oblt-cli/cluster-create-ccs@v1
+        with:
+          remote-cluster: 'dev-oblt'
+          cluster-name-prefix: 'foo'
+          cluster-name-sufix: 'bar'
+          token: ${{ secrets.PAT_TOKEN }}
+
+      # ...
+
+      - uses: elastic/oblt-actions/oblt-cli/cluster-destroy@v1
+        with:
+          cluster-name: 'foo'
```

Comment:
IIUC, the cluster-name in this example should also contain the suffix?

Reply:
Ah okay... 

I'm wondering if it would be technically possible to get the actual cluster name as an output from the cluster-create-* actions, which can then be passed to this.

Reply:
That's one way, another one will be using a fixed name instead.

Ivan documented something about supporting a file:

```yaml

- name: Cleanup
--
85 | if: always()
86 | uses: elastic/apm-pipeline-library/.github/actions/oblt-cli-destroy-cluster@current
87 | with:
88 | cluster-info-file: ${{ env.CLUSTER_INFO_FILE }}
89 | github-token: ${{ env.GITHUB_TOKEN }}


```

Maybe in a follow-up

Reply:
Thinking of it.. I don't actually know the `cluster-name` in the given example.
I copied it from https://github.com/elastic/apm-pipeline-library/blob/main/.github/actions/oblt-cli-destroy-cluster/README.md#usage

In this example the created cluster name should be something l like `foo-<something>-bar`. But what is the value of <something>?

Reply:
We probably need to review the existing implementation in the oblt-cli. For now, I'd say you can move forward, and likely you can change the example snippet with a full-name but without using the create-cluster step to avoid any misleading, what do you think?

Reply:
sounds good. https://github.com/elastic/oblt-actions/pull/37/commits/8fccc124a1ef863022f1e623b693eade287013e6